src/main/java/Poirot.java

```diff
@@ -0,0 +1,204 @@
+import java.io.*;
```

Comment:
Try not to use the `*` to import everything. Instead explicitly import what is needed.